### PR TITLE
quick fixes in editor

### PR DESCRIPTION
**Pull request by Aniket-Engg**
Related to #3927 

This PR introduces QuickFixes for:

- Adding SPDX license
- Adding pragma line

More quick fixes will be added in upcoming PRs

### CODE REVIEW

File: libs/remix-ui/editor/src/lib/providers/codeActionProvider.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { Monaco } from "@monaco-editor/react"
+import monaco from "../../types/monaco"
+import { EditorUIProps } from "../remix-ui-editor"
+import { default as fixes } from "./quickfixes"
+
+export class RemixCodeActionProvider implements monaco.languages.CodeActionProvider {
+    props: EditorUIProps
```
Comment (LianaHus) [inline libs/remix-ui/editor/src/lib/providers/codeActionProvider.ts:7]: Please use one space as a tab

Comment (bunsenstraat) [reply libs/remix-ui/editor/src/lib/providers/codeActionProvider.ts:7]: the default for TS is 4 spaces

Comment (bunsenstraat) [reply libs/remix-ui/editor/src/lib/providers/codeActionProvider.ts:7]: https://github.com/ethereum/remix-project/pull/3939 sets all to 2 spaces. 1 is not best practice.

Comment (Aniket-Engg) [reply libs/remix-ui/editor/src/lib/providers/codeActionProvider.ts:7]: As linting is being fixed for different PR, can you review again? @LianaHus

**Pull request merged**